ui/index_sso.php

```diff
@@ -63,6 +62,15 @@
 
 global $SSO;
 
+if (is_array($SSO) && array_key_exists('ProxyVars', $SSO)) {
+	if (is_bool($SSO['ProxyVars'])) {
```

Comment:
`is_bool($var)` should be changed to `(bool) $var` otherwise `$var` equal false will set support of proxyvars.